AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.txt

```diff
@@ -199,6 +223,7 @@ private func target(_ service: String) -> Target {
             .awsSDKHTTPAuth,
             .awsSDKEventStreamsAuth,
             .awsSDKChecksums,
+            "InternalAWSSTS",
```

Comment:
All services depend on `InternalAWSSTS`, used for credential resolution as needed.